### ISSUE TITLE
Add more features to Access Control Add-on

### ISSUE THREAD

**Issue by njmulsqb** [zaproxy/zaproxy#8356]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

As discussed in https://groups.google.com/g/zaproxy-users/c/OzrzuhEWmHc new features need to be added to Access Control add-on to make it more useful in RBAC testing.

### Describe alternatives you've considered

Autorize, Autorepeater Burp Extensions

### Would you like to help fix this issue?

- [ ] Yes

**Comment by kingthorin** [comment]
Tracker'ified :grinning:

**Comment by kingthorin** [comment]
> but this could be made much simpler by just replacing the cookies of other user. Cookie/auth header can be fed into ZAP by the user manually. This can help with app's having tricky authentication that ZAP cant help with.

That would require the user to ensure (somehow) that the session/cookie didn't invalidate as ZAP would have no way to re-auth.

**Comment by njmulsqb** [comment]
> That would require the user to ensure (somehow) that the session/cookie didn't invalidate as ZAP would have no way to re-auth.

Yes, this doesnt hurt much tbh, while testing I start getting 403 or 302 to login page which shows cookie has expired so I just replace it in autorize. This can be an optional feature on top of current working of auth.

**Comment by kingthorin** [comment]
Added a few more tasks.

**Comment by njmulsqb** [comment]
Added one more task

**Comment by thc202** [comment]
It's already possible to test with unauthenticated user.

**Comment by thc202** [comment]
> Ability to just replace cookies/auth header

This is already possible as well (for cookies at least). You can specify the session from the HTTP Sessions tab for each user.
